SU2_CFD/include/numerics/CNumerics.hpp

```diff
@@ -1316,6 +1444,40 @@ class CNumerics {
    */
   static void tql2(su2double **V, su2double *d, su2double *e, unsigned short n);
 
+  /*!
+   * \brief Set variable indexes for NEMO solver.
+   */
+
+ // inline void SetRhosIndex(unsigned short val_Index) { RHOS_INDEX = val_Index; }
```

Comment:
why did these get commented?

Reply:
because I put them only in the constructor of the NEMO numerical methods. while they were in CNumerics.hpp that meant that these functions would be executed for any solver, but they only serve a purpose for the NEMO solver